src/workerd/io/worker-interface.capnp

```diff
@@ -126,6 +126,31 @@ struct QueueResponse @0x90e98932c0bfc0de {
   # List of Message IDs that were explicitly marked as acknowledged
 }
 
+struct HibernatableWebSocketEventMessage {
+  payload :union {
+    text @0 :Text;
+    data @1 :Data;
+    close :group {
+      code @2 :UInt16;
+      reason @3 :Text;
+      wasClean @4 :Bool;
+    }
+    error @5:Text;
+    # TODO(now): This should be an Exception maybe?
+  }
+}
+
+struct HibernatableWebSocketResponse {
+  delivered @0 :Bool;
```

Comment:
@jqmmes what was this for again? Asking because https://github.com/cloudflare/workerd/pull/343#discussion_r1096348754

Reply:
The original idea was to inform the event dispatcher that the message sent by RPC was delivered, but I guess we don't really need this now